package.json

```diff
@@ -27,10 +27,21 @@
   "devDependencies": {
     "@commitlint/cli": "^18.6.1",
     "@commitlint/config-conventional": "^18.6.2",
+    "@types/chai": "^4",
+    "@types/mocha": "^10",
```

Comment:
are you planning to write some node tests for this too (with mocha) like in #8?
If not, we don't need these deps